src/planwise/client/projects/views.cljs

```diff
@@ -80,6 +89,9 @@
          selected-tab)
         [:div
          [sidebar-section selected-tab]
+         [:div.loading-indicator {:style {:display (if (= @map-state :loading-displayed) "block" "none")}}
```

Comment:
This is an incorrect use of React's model. You should just:

```clojure
(when (= @map-state :loading-displayed)
   [:div.loading-indicator
     [loading-wheel]
     [:span "Retrieving facilities"]])
```

The component should not be rendered (ie. not hidden) if the condition is not met.